Core/Object Arts/Dolphin/Base/DolphinTest.cls

```diff
@@ -77,6 +77,14 @@ runKnownBug: aString test: aSymbol
 	#todo.	"Make this controllable by an option or class var"
 	^false!
 
+setUp
+
```

Comment:
Yes, but it still doesn't belong here. I agree it would be useful if the TestRunner UI updated, but that should be done by observing the TestSuite change events. The UnitTestPlugin does this, so take a look at UnitTestPlugin>>runSuite: to see how to modify the TestRunner to achieve the same effect :-)